src/goalServices/types.ts

```diff
@@ -67,11 +67,21 @@ interface ICourse {
   name: string;
 }
 
+interface ICitation {
+  citation: string;
+  monitoringReferences: JSON;
```

Comment:
would there be value in defining an interface here?

Reply:
What do you mean by this, are you saying the current approach of creating an interface is not needed?

Reply:
JSON is just saying "anything could be in here" isn't it? should we be more specific?